docker/runtime/ruby-2.4/event-trigger/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM bitnami/ruby:2.4
+
+LABEL maintainer "Bitnami <containers@bitnami.com>"
+
+ENV RACK_ENV="production"
+
+RUN install_packages git
```

Comment:
why do you need git here ?

Reply:
Actually not sure, I pretty much copied this over from the http-trigger. @jbianquetti-nami is git really needed?